### DISCUSSION TITLE
Splash screen

### DISCUSSION THREAD

**Discussion by ghost** [canton7/Stylet#338] [Q&A] [answered]
I would add a splash screen to my app, before loading the root window.
So which way should I do?
-make the splash screen window  as the root/main so it started first from bootstrapper, and then navigate it to the other and close it
-from the root hide itself and show the spash screen and then close it and show unhide the root

**Comment by mwdpb** [comment]
Bump up an old thread, I'd also like to know what is the best method to implement Splash window with Stylet.  Thanks in advance!

**Comment by canton7** [comment, marked as answer]
How exactly you do this depends on your exact requirements.

Fundamentally a splash screen is just another bit of navigation. Your ShellView/ShellViewModel starts off displaying the splash screen, then navigates to your main content. Navigation is covered in the documentation.

**Reply by mwdpb** [reply]
Got it, thanks for the quick reply!
